frappe/integrations/doctype/connected_app/connected_app.json

```diff
@@ -96,7 +96,7 @@
   },
   {
    "fieldname": "authorization_uri",
-   "fieldtype": "Data",
+   "fieldtype": "Small Text",
```

Comment:
should we change url fields in same doctype too? 

These aren't required to be searchable so text type is fine.

Reply:
> should we change other url fields in same doctype too?

I don't mind making that change. Not sure if there's any standards for the token, profile and introspection URLs though.

Reply:
> These aren't required to be searchable so text type is fine.

@ankush, `Small Text` fields have visually smaller boxes than `Text` fields. And since URL length is capped at 2083 characters, I think we should be fine.

Reply:
I meant sql `text` type. UI doesn't matter much in this case.

Reply:
Ah okay lol. I'll avoid changing the datatypes for the other URLs at this point though. We can pick those up as they go.